changes/2284.feature.rst

```diff
@@ -0,0 +1 @@
+Added improved document types (file associations) for Windows and macOS platforms
```

Comment:
Since 0.3.23 has been released, this feature is only macOS updates now.
```suggestion
Improved document types and file associations for macOS platforms.
```